example-site/content-org/construct-hugo-front-matter-from-menu-meta-data.org

```diff
@@ -0,0 +1,12 @@
+#+HUGO_BASE_DIR: ../
+#+HUGO_FRONT_MATTER_FORMAT: yaml
+
+* Menu Meta Data in YAML Front Matter
+:PROPERTIES:
+:EXPORT_FILE_NAME: menu-meta-data-yaml
+:EXPORT_DATE: 2017/07/17
+:HUGO_MENU_NAME: main
+:HUGO_MENU_WEIGHT: 100
+:HUGO_MENU_PARENT: posts
```

Comment:
I think there's a problem with this schema. SUppose I have a large org file with many posts.  Most of the tie I just let the menu be constructed with default values for all menu attributes. However, for a few posts I want to change the menu weight (the posts are important and I want them to come at the top of the menu).  In order to make that one change, I have to correctly transcribe all the menu values and change only the one I care about.  

The old schema (as written in my commit) is a bit clumsy but maybe it's ultimately more flexible for the end-user.